### ISSUE TITLE
React on Rails is logging in production

### ISSUE THREAD

**Issue by justin808** [shakacode/react_on_rails#829]
![2017-05-01_16-52-26](https://cloud.githubusercontent.com/assets/1118459/25602324/9aced764-2e8e-11e7-8781-f50dbb0fa8c1.png)

https://github.com/shakacode/react_on_rails/blob/master/spec/dummy/config/initializers/react_on_rails.rb

```ruby
  # default is true for development, off otherwise
  config.trace = Rails.env.development?
```

**Comment by Conturbo** [comment]
i'm gonna take a crack at it! I'll let you know if i get stuck.

**Comment by justin808** [comment]
Fixed by #845.
